data/parts/ChartStylePart.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Root": "cs:CT_ChartStyle",
+  "Name": "ChartStylePart",
+  "Base": "OpenXmlPart",
+  "ContentType": "application/vnd.ms-office.chartstyle\u002Bxml",
```

Comment:
this is probably something I'll have to look into: why the unicode char in the content type. I don't think those are in what Office emits but I could be wrong. not really an action for you @twsouthwick

Reply:
I can confirm the JSON encoding thing. It looks like you are using `System.Text.Json`, which encodes more characters than `Newtonsoft.Json` when serializing. For example, `Newtonsoft.Json` does not encode `+` characters as `\u002B` (at least not by default) while `System.Text.Json` does. However, both will correctly deserialize the serialized value:

- `"application/vnd.ms-office.chartstyle\u002Bxml"`

into:

- `"application/vnd.ms-office.chartstyle+xml"`.

Reply:
Again, thanks Taylor and Thomas for the explanations.